index.js

```diff
@@ -326,7 +336,7 @@ NYC.prototype.clearCache = function () {
 }
 
 NYC.prototype.createTempDirectory = function () {
-  mkdirp.sync(this.tempDirectory())
+  mkdirp.sync(this.processInfoDirectory())
```

Comment:
could we make it so we only create this directory, if we've enabled process tracking; my thinking is, let's make the behavior 100% identical, unless you are outputting the process tree.